test/jdk/javax/sound/midi/Devices/OutputBuffer.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import javax.sound.midi.*;
+
+/**
+ * @test
+ * @key sound
+ * @bug 8074211 8250667
+ * @summary Test if part of the previous sysex message in the output buffer
+ *          is sent again with the next sysex message
+ * @comment This test does not fail when the bug occurs. It requires manual
+ *          monitoring of the output.
```

Comment:
The fix seems reasonable. but this test comment is odd. I'm not sure I understand what you can mean by manual monitoring of the output. The test doesn't seem useful unless it can detect failure.

Reply:
The test is useful since that is the only test that touches the code path when the buffer for the midi message is reused and the bug can be reproduced. at least it doesn't crash.

Reply:
Actually I noticed the bug when I was working on https://github.com/openjdk/jdk/pull/16477. The testcase I added in the PR also triggered the bug. However, the testcase crashes JVM before triggering the bug, so I use another testcase for this PR.

The test case sends the folloing three pieces of sysex message
```
F0 7D 01 02
F7 03 04
F7 05 06 07 F7
```

The expected message received should be the following
```
F0 7D 01 02 03 04 05 06 07 F7
```

But the acual message received is
```
F0 7D 01 02 03 04 01 02 05 06 07 F7
```

Therefore, I'm not sure whether it's better to remove the testcase in this PR or keep it.

Reply:
I removed the testcase and reuse the testcase added in #16477.